pkgs/os-specific/linux/rdma-core/default.nix

```diff
@@ -4,7 +4,7 @@
 } :
 
 let
-  version = "33.0";
+  version = "33.1";
```

Comment:
that let in could be removed here.

Reply:
I prefer the let pattern over a recursive set. What would be the motivation to change it?

Reply:
I don't think that there is any reason to prefer let in here. It adds 2 more lines than necessary and you can just declare the version in the derivation with a little rec instead of adding a let in where you write version two times.

Reply:
I still think it is cleaner than a `rec` here (see also https://nix.dev/anti-patterns/index.html). The two more lines do not hurt either.